teamcity/build_type.go

```diff
@@ -263,7 +263,7 @@ func (s *BuildTypeService) GetByID(id string) (*BuildType, error) {
 // Rename changes the name of a build type by sending a PUT request
 func (s *BuildTypeService) Rename(id string, name string) error {
 	locator := LocatorID(id).String()
-	_, err := s.restHelper.putTextPlain(locator + "/name", name, "build type name")
+	_, err := s.restHelper.putTextPlain(locator+"/name", name, "build type name")
```

Comment:
I might leave this out until we have to change this function, just to avoid its effect on `git blame`

Reply:
go fmt did this. I can prevent go fmt from running on it if you'd prefer.